core/shared/src/main/scala/org/polyvariant/colorize.scala

```diff
@@ -50,12 +50,23 @@ object colorize {
 
     def renderConfigured(config: RenderConfig): String = {
 
-      def go(self: ColorizedString, currentColors: List[String]): String =
+      val render: (Color, String) => String =
+        config.mode match {
+          case ColorMode.Ansi =>
+            (color, text) =>
+              color match {
+                case Color.Ansi(prefix) => prefix + text + config.resetString
+                case _                  => text
```

Comment:
This approach is not ideal. 

It does not make sense to render char sequences not supported by the platform. On the other hand, the behavior can be non-intuitive.

Reply:
I think it's OK to omit the unsupported sequence. This enables us to fall back to an ANSI color (if it's applied second): `"R".rgb(255, 0, 0).red`

Reply:
However, if we go with the "make your own colorize" option, we can make it a throwing operation when you use `.rgb` on a colorize that doesn't support it. Or even encode RGB support in the types, so that `.rgb` just won't be available if your color mode is ANSI.

Reply:
okay, I think making it typed is a bit overkill, it's probably fine to ignore RGB in ansi mode 😅